### ISSUE TITLE
Feature request: Open Notepad++ via command line with a certain character set

### ISSUE THREAD

**Issue by cyclingcat** [notepad-plus-plus/notepad-plus-plus#8082]
Hi there,

following [this discussion](https://community.notepad-plus-plus.org/topic/19133/open-notepad-via-command-line-with-a-certain-character-set) I'd like to propose a command line option that indicates the character set and/or encoding when opening a file, e.g.:

`notepad++ [filepath[#encoding]]`

Example:

`notepad++ c:\mylogfiles\robocopy.log#OEM850`

(to open the mentioned file with "OEM 850" encoding without having to change the encoding manually via the corresponding menu item.)

The cycling cat

**Comment by datatraveller1** [comment]
I also need this urgently.

**Comment by jojastahl** [comment]
Often, it is not possible to add a postfix to the filename. For example if you configure notepad++ as your git editor. git will append the filename to the configured command line and as far as I know you can't force the filename postfix.

Therefore I would prefer a real option `-encoding utf-8`, or support both.

Also note that `#` is a valid char in filenames, probably rarely used, and a comment initiator in scripts. Something like a `:` may be better?

**Comment by alankilborn** [comment]
>I would prefer a real option -encoding utf-8

Yes, makes sense.  I said this before, see [HERE](https://community.notepad-plus-plus.org/topic/19133/open-notepad-via-command-line-with-a-certain-character-set) and look for Nov 6, 2022, 7:30 AM posting.

------------------------

>or support both.

That would be too much work for the developers, and would introduce a wacky syntax into the command line.  Not advisable.
